app/1.0/blog/2016-10-03-1.7-release.md

```diff
@@ -0,0 +1,25 @@
+---
+title:  "1.7 Release"
+
+---
+_We announce significant releases here on the blog, highlighting the major new features as well as breaking changes. As always, we'd love to get your feedback, issues, and PR's, so tweet us [@polymer](https://twitter.com/polymer) or contribute to the [Github](https://github.com/Polymer)._
+
+We're pleased to announce the release of:
+
+## Polymer 1.7
+
+The primary goal of the upcoming Polymer 2.0 release is to take advantage of native, cross-browser support for Web Components. With this in mind, Polymer 1.7.0 provides forwards compatibility and initial support for migration. 
+
+### `<slot>` element 
+
+Elements can opt-in to using the new `<slot>` element and `::slotted()` CSS selector from ShadowDOM v1.
```

Comment:
If possible, it would be super nice to link to a doc paragraph or something that lets you read more about this. Same with the next item. I know we link out to the full release notes but it would be good to save folks time navigating through that.